.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Run tests
```

Comment:
The Test workflow will run when the Build workflow completes AND is successful. It installs the artifact from the build step and tests on mac/win/linux for py 3.8 through 3.11.